src/ray/object_manager/ownership_based_object_directory.cc

```diff
@@ -208,7 +209,7 @@ ray::Status OwnershipBasedObjectDirectory::LookupLocations(
     RAY_LOG(WARNING) << "Object " << object_id << " does not have owner. "
                      << "LookupLocations returns an empty list of locations.";
     io_service_.post([callback, object_id]() {
-      callback(object_id, std::unordered_set<NodeID>(), "");
+      callback(object_id, std::unordered_set<NodeID>(), "", NodeID::Nil());
```

Comment:
It looks like the ownership based object directory didn't implement spilling yet.